Spp 002 nl (#157)
inbo/protocolsource: source/spp/mnm/spp_002_nl_locatie-eval_mnm/04_werkwijze.Rmd

Ga naar de te monitoren locatie.
Volg subprotocol A `<protocol-code>_YYYY.NN` om te bepalen of de locatie tot de doelpopulatie die men wil monitoren behoort (eerste bezoek) of nog steeds behoort (herhaald bezoek).
Volg subprotocol B `<protocol-code>_YYYY.NN` waarin de staalnameprocedure wordt uitgelegd.
Volg subprotocol C `<protocol-code>_YYYY.NN` waarin labo-analyse wordt uitgelegd.

De doelstelling is om de subprotocols te organiseren in een coherent hoofd protocol waarin ontbrekende elementen aangevuld worden die niet in de subprotocols staan, zoals wanneer een bepaald subprotocol wel of niet wordt uitgevoerd, of kleine extra stappen die geen eigen subprotocol vereisen.
-->
```

## Overzicht van de stappen

De steekproef van terrestrische steekproefeenheden, aangeleverd door de meetnetontwerpketen, wordt beschikbaar gesteld.
Via GIS-applicaties (vb. QGIS en QField) worden ze gevisualiseerd aangeboden voor desktopscreening en voor de communicatie van gegevens voor de organisatie op terrein (*outbound*) en de op terrein verzamelde gegegevens (*inbound*).

De evaluatie van een terrestrische steekproefeenheid gebeurt eerst door een desktopscreening (zie \@ref(desktop)):

> - Indien uit de desktopscreening blijkt dat het doeltype van een steekproefeenheid en zijn vervangeenheden er niet aanwezig kan zijn, dient de steekproefeenheid te worden verwijderd uit de steekproef.

Reviewer: ```suggestion
- Indien uit de desktopscreening blijkt dat het doeltype van een steekproefeenheid en zijn vervangeenheden er niet aanwezig kan zijn, dient dit aangeduid te worden zodat de steekproefeenheid kan worden verwijderd uit de steekproef.
```

> - Indien een steekproefeenheid of zijn vervangeenheden het doeltype kan bevatten op basis van de visuele controle, wordt de steekproefeenheid bezocht om ter plekke te beoordelen of het aanwezige type (habitat(sub)type of rbb) overeenkomt met het doeltype waarvoor de steekproefeenheid werd getrokken (zie \@ref(terplekke)).

Reviewer: ```suggestion
- Indien een steekproefeenheid of zijn vervangeenheden het doeltype wel kan bevatten op basis van de visuele controle, wordt de steekproefeenheid bezocht om ter plekke te beoordelen of het aanwezige type (habitat(sub)type of rbb) overeenkomt met het doeltype waarvoor de steekproefeenheid werd getrokken (zie \@ref(terplekke)).
```

> - Indien een steekproefeenheid of zijn vervangeenheden het doeltype kan bevatten op basis van de visuele controle, wordt de steekproefeenheid bezocht om ter plekke te beoordelen of het aanwezige type (habitat(sub)type of rbb) overeenkomt met het doeltype waarvoor de steekproefeenheid werd getrokken (zie \@ref(terplekke)).

Reviewer: ```suggestion
- Indien een steekproefeenheid of zijn vervangeenheden het doeltype kan bevatten op basis van de desktopscreening, wordt de steekproefeenheid bezocht om ter plekke te beoordelen of het aanwezige type (habitat(sub)type of rbb) overeenkomt met het doeltype waarvoor de steekproefeenheid werd getrokken (zie \@ref(terplekke)).
```

The evaluation at the location is also visual, isn't it?

  - Als het actuele habitat(sub)type of rbb ter plekke overeenkomt met het beoogde doeltype waarvoor de steekproefeenheid werd getrokken, wordt aansluitend een celkartering uitgevoerd (zie \@ref(celkartering)).

>   - Als het niet overeenkomt met het doeltype, wordt aansluitend gezocht naar een vervanging door een geschikte rastercel in dezelfde polygoon (zie \@ref(lokalevervanging)).

Reviewer: ```suggestion
  - Als het actuele type niet overeenkomt met het doeltype, wordt aansluitend gezocht naar een vervanging door een geschikte rastercel in dezelfde polygoon (zie \@ref(lokalevervanging)).
```

>   - Als het niet overeenkomt met het doeltype, wordt aansluitend gezocht naar een vervanging door een geschikte rastercel in dezelfde polygoon (zie \@ref(lokalevervanging)).

Reviewer: ```suggestion
  - Als het niet overeenkomt met het doeltype, wordt aansluitend gezocht naar een vervanging door een geschikte rastercel in dezelfde polygoon (onderdeel) (zie \@ref(lokalevervanging)).
```

    - Indien een geschikte rastercel gevonden wordt, zal deze de originele steekproefeenheid vervangen (maar beide GRTS-rangnummers worden behouden in de databank). Aansluitend wordt een celkartering uitgevoerd in de vervangcel

>     - Indien geen geschikte vervangcel gevonden wordt, is geen lokale vervanging mogelijk en wordt de steekproefeenheid verwijderd uit de steekproef.

Reviewer: ```suggestion
    - Indien geen geschikte vervangcel gevonden wordt, is geen lokale vervanging mogelijk en wordt de steekproefeenheid op een later moment / als gevolg hiervan verwijderd uit de steekproef.
```
To make clear that deleting sampling units is not part of this protocol.

> In de eerste twee stappen wordt beoordeeld of het doeltype kan aanwezig zijn en ter plekke actueel aanwezig is in het celmidden of ergens in de cel van de steekproefeenheid.

Reviewer: Which first two steps?

> Doeltypes die *ergens* in de cel dienen voor te komen, zijn:

Reviewer: ```suggestion
Doeltypes die *ergens* in de cel dienen voor te komen (i.e., cellgekoppelde types), zijn:
```

- habitattypes: 1320, 2150, 6210_sk, 6430_bz en 7150
- rbb‐types: brem‐ en gaspeldoornstruweel (rbbsg) en doornstruweel (rbbsp).

> Alle andere terrestrische doeltypes (met uitzondering van habitattype 7220) dienen aanwezig te zijn in het celmidden.

Reviewer: ```suggestion
Alle andere terrestrische doeltypes (met uitzondering van habitattype 7220) dienen aanwezig te zijn in het celmidden (i.e., cellgekoppelde types).
```

> Voor de kalktufbronnen (habitattype 7220) is de populatie‐eenheid de volledige habitatvlek (met variabele, doorgaans beperkte oppervlakte), geïdentificeerd met een punt gekoppeld aan de vlek.

Reviewer: Population unit (vs. sampling unit) has not been defined yet - not sure it's an issue.

De evaluatie van dit doeltype gebeurt dus niet op niveau van rastercellen maar de hele vlek en valt dus buiten het bestek van voorliggend protocol.

> **Let op**: een steekproefeenheid kan voor meer dan één doeltype zijn getrokken.

Reviewer: ```suggestion
**Let op**: een steekproefeenheid en dus rastercel kan voor meer dan één doeltype zijn getrokken.
```

> Dit is het gevolg van het feit dat een polygoon in de Habitatkaart, waarop de waarden van de rastercellen zijn gebaseerd, meer dan één type (habitat of rbb) kan bevatten, in relatieve verhoudingen zoals weergegeven met de zogenaamde phab-waarden.

Reviewer: ```suggestion
Dit is het gevolg van het feit dat een polygoon in de Habitatkaart, waarop de waarden van de rastercellen zijn gebaseerd, meer dan één celgekoppelde type (habitat of rbb) kan bevatten, in relatieve verhoudingen zoals weergegeven met de zogenaamde phab-waarden.
```

Reviewer: ```suggestion
Dit is het gevolg van het feit dat een polygoon in de Habitatkaart, waarop de waarden van de rastercellen zijn gebaseerd, meer dan één type (habitat of rbb) kan bevatten, in relatieve verhoudingen zoals weergegeven met de phab-waarden.
```

phab- values have already been introduced

> Dit is het gevolg van het feit dat een polygoon in de Habitatkaart, waarop de waarden van de rastercellen zijn gebaseerd, meer dan één type (habitat of rbb) kan bevatten, in relatieve verhoudingen zoals weergegeven met de zogenaamde phab-waarden.

Reviewer: ```suggestion
Dit is het gevolg van het feit dat een polygoon in de Habitatkaart, waarop de waarden van de rastercellen zijn gebaseerd, meer dan één type (habitat of rbb) kan bevatten, in relatieve verhoudingen zoals weergegeven met de zogenaamde phab-waarden. Zo kan een steekproefeenheid meerdere celgekoppelde types bevatten, al dan niet in combinatie met een celmiddengekoppelde type.
```
move below sentence up here

> In het celmidden zal er echter in realiteit normaliter maar één type voorkomen en dus sluiten types die celmiddengebaseerd zijn elkaar uit.

Reviewer: ```suggestion
In het celmidden zal er echter in realiteit normaliter maar één type voorkomen en dus sluiten types die celmiddengekoppeld zijn elkaar uit.
```
to avoid confusion with "cellgebaseerd" (check throughout the protocol as this seems a returning issue).

> Wel kan een steekproefeenheid meerdere celgebaseerde types bevatten, al dan niet in combinatie met een celmiddengebaseerd type.

Reviewer: ```suggestion

```
suggestion to move up

> Wel kan een steekproefeenheid meerdere celgebaseerde types bevatten, al dan niet in combinatie met een celmiddengebaseerd type.

Reviewer: ```suggestion
Wel kan een steekproefeenheid meerdere celgekoppelde types bevatten, al dan niet in combinatie met een celmiddengekoppeld type.
```

> Check dus bij de evaluatie van een terrestrische steekproefeenheid of je deze procedure moet doorlopen voor meer dan één doeltype binnen hetzelfde jaar, dan kan je het werk hiervoor combineren in één procedure.

Reviewer: Whose task is this?

> Check dus bij de evaluatie van een terrestrische steekproefeenheid of je deze procedure moet doorlopen voor meer dan één doeltype binnen hetzelfde jaar, dan kan je het werk hiervoor combineren in één procedure.

Reviewer: ```suggestion
Check dus bij de evaluatie van een terrestrische steekproefeenheid of je deze procedure moet doorlopen voor meer dan één celgekoppeld doeltype binnen hetzelfde jaar, dan kan je het werk hiervoor combineren in één procedure.
```

> ## Desktopscreening van steekproefeenheden en zijn vervangeenheden {#desktop}

Reviewer: Maybe add a few words regarding the implementation of this screening (i.e., the application available to do this; maybe also a reference to a documentation of the application - although it is possibly better to add this to section 4?)

> Om grote, duidelijke fouten in de steekproef op te sporen, dient een steekproefeenheid en zijn steekproefeenheden onderworpen te worden aan een eerste visuele screening.

Reviewer: "visuele screening" - specify: "GIS-gebaseerd"? "op afstand"?
(see an earlier comment)

> Hiervoor dient de rastercel van de steekproefeenheid ruimtelijk gevisualiseerd te worden (vb. in QGIS) in combinatie met een aantal GIS-lagen in de meest recente versie, zoals orthofoto's (zomer- en winteropnames), de Habitatkaart, de Watervlakkenlaag en het Digitaal Hoogtemodel Vlaanderen.

Reviewer: include reference for the "Digitaal Hoogtemodel Vlaanderen"?

> Toets visueel af of de rastercel van de steekproefeenheid het doeltype kan bevatten zoals vermeld in databank.

Reviewer: ```suggestion
Toets visueel af of de rastercel van de steekproefeenheid het doeltype kan bevatten zoals vermeld in databank met steekproefeenheden.
```

> Voorbeelden van fouten die hiermee kunnen worden opgespoord zijn: recent verdwenen habitat door ruimtebeslag, recent gewijzigd habitat door beheer, open doelhabitat gelegen in boshabitat of omgekeerd en rastercellen van celmiddengebaseerde types die slechts in een klein hoekpunt overlappen met een polygoon van het doelhabitat.

Reviewer: ```suggestion
Voorbeelden van fouten die hiermee kunnen worden opgespoord zijn: recent verdwenen habitat door ruimtebeslag, recent gewijzigd habitat door beheer, open doelhabitat gelegen in boshabitat of omgekeerd en rastercellen van celmiddengekoppelde types die slechts in een klein hoekpunt overlappen met een polygoon van het doelhabitat.
```

> Duid aan in de databank met steekproefeenheden of de steekproefeenheid volgens deze screening wel of niet tot het doeltype kan behoren.

Reviewer: ```suggestion
Duid aan in de databank of de steekproefeenheid volgens deze screening wel of niet tot het doeltype kan behoren.
```

> Als het er niet toe kan behoren, controleer de vervangeenheden.

Reviewer: ```suggestion
Als het er niet toe kan behoren, controleer de vervangeenheden en geef ook hier negatieve beoordelingen aan.
```

Het resultaat van de screening wordt gebruikt om de steekproef bij te werken: indien de steekproefeenheid en zijn vervangeenheden **overduidelijk** niet tot het doeltype kunnen behoren, wordt het verwijderd uit de steekproef.

> Indien er geen duidelijke visuele aanwijzingen zijn dat het doeltype er niet kan aanwezig zijn, wordt de steekproefeenheid in een volgende stap beoordeeld op de aanwezigheid van het doeltype ter plekke.

Reviewer: "steekproefeenheid (of vervangeenheid)"?

If screening is negative for sampling but positive for replacement unit?

## Beoordeling op aanwezigheid van het doeltype ter plekke {#terplekke}

### Bereid het bezoek aan de steekproefeenheid voor

> Controleer in de GIS-applicatie wie de eigenaar en/of beheerder is van het gebied waarin de steekproefeenheid gelegen is.

Reviewer: Specify that (if) this is the MNM-specific application.

Toets af bij de meetnetverantwoordelijke of projectcoördinator of vergunningen noodzakelijk zijn en je bezoek is gemeld bij de eigenaar en/of beheerder.
Toegankelijkheid van militaire gebieden dient door de uitvoerder van de locatie-evaluatie een week voor het terreinbezoek aangevraagd te worden via mail aan het betreffende kwartier, met de boswachter in cc.
Zij laten weten op welke dagen en tussen welke uren je het terrein mag betreden.
Bij het betreden en het verlaten van het terrein op de dag zelf dien je telkens telefonisch contact op te nemen met de kwartiercommandant.

### Evalueer de timing van je bezoek

Evalueer of je geplande bezoek gebeurt in de (sub)optimale periode voor kartering van het doeltype.

<!-- nog tabel invoegen met optimale periode -->

### Evalueer de toegankelijkheid en bezoek de steekproefeenheid

Indien het terrein, waar de steekproefeenheid gelegen is, ontoegankelijk is, duid dit aan in de GIS-applicatie en vermeld de reden en de vermoedelijke duur van ontoegankelijkheid.

> Mogelijke redenen zijn: overstroming, afsluiting, broedseizoen, beheerwerken, enz.

Reviewer: "enz."
more examples of what this can (not) be?
A list is best as exhaustive as possible to avoid individual interpretations.

Is het terrein wel toegankelijk, navigeer je naar de steekproefeenheid.

### Bezoek de steekproefeenheid en navigeer naar het celmidden

Met de hulp van de GPS-ontvanger op je tablet of smartphone navigeer je naar de locatie van de te beoordelen steekproefeenheid.
Eens je je in de nabijheid van de meetlocatie bevindt (vb. op 10 m), navigeer je met behulp van de RTK-GPS-ontvanger naar de exacte locatie van het centrum van de steekproefeenheid.

**Let op**: in de rastercel kan een permanent kwadraat van het meetnet biotische habitatkwaliteit aanwezig zijn.
Het is 3 m op 3 m groot en bevindt zich ten noordwesten van het celmidden: het celmidden van de rastercel is het zuidoostelijk hoekpunt van het permanent kwadraat.
<!-- nog figuur invoegen met kwadraat in rastercel --> Vermijd zo veel mogelijk dit kwadraat om vertrappeling van de vegetatie te voorkomen.
Dit kan eenvoudig door het celmidden te benaderen vanuit het zuidoosten, indien het terrein dit toelaat.

> ### Evalueer of het doeltype aanwezig is {#doeltype}

Reviewer: The current structure of this section is not entirely clear to me; I added some suggestions to render it more linear. It seems to be going a bit back and forth at the moment.

> De GIS-applicatie geeft aan welk doeltype op een steekproefeenheid aanwezig zou moeten zijn en indien van toepassing de laatste keer dat het type werd bepaald in het veld.

Reviewer: ```suggestion
De GIS-applicatie geeft aan welk doeltype(s) op een steekproefeenheid aanwezig zou moeten zijn en indien van toepassing de laatste keer dat het type werd bepaald in het veld.
```
for cell-coupled types?

> De GIS-applicatie geeft aan welk doeltype op een steekproefeenheid aanwezig zou moeten zijn en indien van toepassing de laatste keer dat het type werd bepaald in het veld.

Reviewer: ```suggestion
De GIS-applicatie geeft aan welk doeltype op een steekproefeenheid aanwezig zou moeten zijn en indien van toepassing de laatste keer dat de aanwezigheid van het type werd bepaald in het veld.
```

De wijze waarop het wordt geëvalueerd is afhankelijk van het doeltype:

> - voor de evaluatie van de aanwezigheid van habitat(sub)types 1320, 2150, 6210_sk, 6430_bz en 7150 en de rbb‐types brem‐ en gaspeldoornstruweel (rbbsg) en doornstruweel (rbbsp) is het voldoende dat het doeltype ergens in de 32 m x 32 m rastercel aanwezig is.

Reviewer: ```suggestion
- voor de evaluatie van de aanwezigheid van de cellgekoppelde types (habitat(sub)types 1320, 2150, 6210_sk, 6430_bz en 7150 en de rbb‐types brem‐ en gaspeldoornstruweel (rbbsg) en doornstruweel (rbbsp)) is het voldoende dat het doeltype ergens in de 32 m x 32 m rastercel aanwezig is.
```

> - voor alle andere terrestrische types (behalve 7220) dient het doeltype aanwezig te zijn in het midden van de rastercel.

Reviewer: ```suggestion
- voor alle andere terrestrische types (behalve 7220) dient het doeltype aanwezig te zijn in het midden van de rastercel (cellmiddengekoppelde types).
```

> De bepaling van het aanwezige habitat(sub)type of rbb gebeurt met behulp van de [veldsleutels](https://www.vlaanderen.be/inbo/de-biologische-waarderingskaart/bwk-handleiding-veldsleutels/).

Reviewer: ```suggestion

```

> Het doeltype moet op minstens 9 m^2^ aanwezig zijn binnen de rastercel.

Reviewer: ```suggestion

```

> Door veranderingen op het terrein, onnauwkeurigheden in de afbakening van de types of het voorkomen van complexen in de karteringen kan het in de praktijk voorkomende type afwijken van het type op de Habitatkaart.

Reviewer: refer to earlier section where this is explained in more detail

> Bijgevolg dienen we steeds het type te bepalen met behulp van de veldsleutels.

Reviewer: ```suggestion
Bijgevolg dienen we steeds het aanwezige habitat(sub)type of rbb te bepalen met behulp van de [veldsleutels](https://www.vlaanderen.be/inbo/de-biologische-waarderingskaart/bwk-handleiding-veldsleutels/).
```

> Het schaalniveau waarop de sleutel dient toegepast te worden, is niet dat van het celmidden, maar deze van de min of meer homogene vegetatievlek waarin het celmidden zich bevindt.

Reviewer: Suggestion to either integrate lower if this is specific for cel center-coupled types or make explicit that (and why) this holds for cell-based types in general.

> Het schaalniveau waarop de sleutel dient toegepast te worden, is niet dat van het celmidden, maar deze van de min of meer homogene vegetatievlek waarin het celmidden zich bevindt.

Reviewer: ```suggestion
Het schaalniveau waarop de sleutel dient toegepast te worden, is niet dat van het celmidden, maar deze van de min of meer homogene vegetatievlek waarin het celmidden zich bevindt. Het doeltype moet hierbij op minstens 9 m^2^ aanwezig zijn binnen de rastercel.
```
Would it make sense to move this here?

> Om de finale beslissing te maken of het punt wordt meegenomen in de steekproef, wordt een denkbeeldig 3 m x 3 m-vlak beoordeeld.

Reviewer: ```suggestion
Om de finale beslissing te maken of het punt wordt meegenomen in de steekproef, wordt dus een denkbeeldig 3 m bij 3 m-vlak beoordeeld.
```

> In het geval de locatie-evaluatie moet worden uitgevoerd niet enkel voor de MNM maar ook voor een LSVI-bepaling (steekproef) dient dient het het 3 m bij 3 m vlak ten noordwesten van het celmidden beoordeeld te worden.

Reviewer: ```suggestion
In het geval de locatie-evaluatie moet worden uitgevoerd niet enkel voor de MNM maar ook voor een LSVI-bepaling (steekproef) dient het 3 m bij 3 m vlak ten noordwesten van het celmidden beoordeeld te worden.
```

> In het geval de locatie-evaluatie moet worden uitgevoerd niet enkel voor de MNM maar ook voor een LSVI-bepaling (steekproef) dient dient het het 3 m bij 3 m vlak ten noordwesten van het celmidden beoordeeld te worden.

Reviewer: what does "(steekproef)" refer to?

**Onderstaande beslisregels worden gehanteerd voor het bepalen of de steekproefeenheid geschikt is.**

> - Voor de celmiddengebaseerde types: het celmidden van de rastercel maakt deel uit van een homogene, grotere, aaneengesloten vegetatievlek van het doeltype (minimum 9 m^2^).

Reviewer: ```suggestion
- Voor de celmiddengekoppelde types: het celmidden van de rastercel maakt deel uit van een homogene, grotere, aaneengesloten vegetatievlek van het doeltype (minimum 9 m^2^).
```

> - Voor de celgebaseerde types: de rastercel bevat de minimumoppervlakte van het doeltype.

Reviewer: ```suggestion
- Voor de celgekoppelde types: de rastercel bevat de minimumoppervlakte van het doeltype.
```

Geef in de GIS-applicatie de volgende gegevens in:

- je naam
- de datum waarop je deze evaluatie uitvoert
- het actueel aanwezige habitat(sub)type of rbb in de cel of in het celmidden

> - eventuele opmerkingen

Reviewer: e.g.?

Is de steekproefeenheid **niet geschikt**, start aansluitend met de vervangprocedure (zie \@ref(lokalevervanging)).

## Lokale vervanging van niet-geschikte steekproefeenheden {#lokalevervanging}

Blijkt, na het doorlopen van bovenstaande stappen, dat de steekproefeenheid **het doeltype niet bevat**, dan doorloop je aansluitend de procedure voor het vinden van een geschikte vervangeenheid.
Je gaat hierbij in (een deel van) de polygoon waarin de niet-geschikte steekproefeenheid gelegen is, op zoek naar een vervangeenheid die wel het doeltype bevat.
Volg hiervoor het protocol [sfp-001-nl 2023.01](https://inbo.github.io/protocols/2023.01/index.html).
In de GIS-applicatie vind je voor elke te evalueren steekproefeenheid een lijst van mogelijke vervangeenheden die je volgens een vooraf vastgelegde volgorde evalueert zoals beschreven in protocol [sfp-001-nl 2023.01](https://inbo.github.io/protocols/2023.01/index.html).

Aanvullend op dat protocol geldt voor lokale vervangingen in grote polygonen een extra regel.
In het geval van grote polygonen kan het aflopen en evalueren van de mogelijke vervangeenheden zeer veel moeite en tijd kosten.

> Om dit te beperken, worden grote polygonen opgedeeld volgens de met de polygoon overlappende GRTS-level03-cellen van 128 m op 128 m.

Reviewer: "GRTS-level03-cellen" simplify in analogy to the earlier removal of this kind of information?
Something like: 
"sets / kwadraten van GRTS rastercellen"/"grotere GRTS rastercellen"...

> Enkel die GRTS-level03-cel waarin de originele steekproefeenheid gelegen is (én meer dan ... rastercellen bevat), wordt verder beschouwd.

Reviewer: "..." ?

Hierdoor knelt het meetmerk vast in het buisanker.
Controleer of het anker vast zit in de grond: als je het gemakkelijk kan uittrekken, plaats dan een langer buisanker.
Staat er hoge, dichte vegetatie waardoor je het meetmerk moeilijk zou terugvinden (vb. bramenstruweel, riet), plaats een bamboestok met gekleurde top naast het meetmerk.

<!-- nog foto invoegen van meetmerk met bamboestok -->

### Oriënteer jezelf en de rastercel op het terrein

Oriënteer jezelf op het terrein met behulp van de locatie van het celmidden, de visuele voorstelling van de rastercel en de Habitatkaart in de GIS-applicatie.
Recente orthofoto's kunnen ook hierbij helpen.
Je kan ook navigeren naar (één van) de vier hoekpunten van de rastercel om het gebied binnen de rastercel af te bakenen.
Gebruik eventueel 4 bamboestokken om de hoekpunten te markeren

### Beoordeel het gebied binnen de rastercel en markeer de celkarteringspolygoon

Voer de celkartering uit: teken in de GIS-applicatie met behulp van de RTK-GPS-ontvanger de grenzen van de gebieden in de rastercel waar het doeltype actueel aanwezig is.

> Dit kan door de rastercel systematisch te doorlopen (vermijd de zone van het permanent kwadraat) of aan de rand van de rastercel voor zover de gevoeligheid van de vegetatie dit toelaat.

Reviewer: ```suggestion
Dit kan door de rastercel systematisch te doorlopen (vermijd de zone van het permanent kwadraat van het meetnet biotische habitatkwaliteit) of aan de rand van de rastercel voor zover de gevoeligheid van de vegetatie dit toelaat.
```

> Dit kan door de rastercel systematisch te doorlopen (vermijd de zone van het permanent kwadraat) of aan de rand van de rastercel voor zover de gevoeligheid van de vegetatie dit toelaat.

Reviewer: What is meant by "aan de rand van de rastercel"?

De evaluatie van een terrestrische steekproefeenheid gebeurt eerst door een desktopscreening (zie \@ref(desktop)):

-   Indien uit de desktopscreening blijkt dat het doeltype van een steekproefeenheid en zijn vervangeenheden er niet aanwezig kan zijn, dient de steekproefeenheid te worden verwijderd uit de steekproef.

-   Indien een steekproefeenheid of zijn vervangeenheden het doeltype kan bevatten op basis van de visuele controle, wordt de steekproefeenheid bezocht om ter plekke te beoordelen of het aanwezige type (habitat(sub)type of rbb) overeenkomt met het doeltype waarvoor de steekproefeenheid werd getrokken (zie \@ref(terplekke)).

    -   Als het actuele habitat(sub)type of rbb ter plekke overeenkomt met het beoogde doeltype waarvoor de steekproefeenheid werd getrokken, wordt aansluitend een celkartering uitgevoerd (zie \@ref(celkartering)).

    -   Als het niet overeenkomt met het doeltype, wordt aansluitend gezocht naar een vervanging door een geschikte rastercel in dezelfde polygoon (zie \@ref(lokalevervanging)).

>         -   Indien een geschikte rastercel gevonden wordt, zal deze de originele steekproefeenheid vervangen (maar beide GRTS-rangnummers worden behouden in de databank). Aansluitend wordt een celkartering uitgevoerd.

Reviewer 2: ```suggestion
        -   Indien een geschikte rastercel gevonden wordt, zal deze de originele steekproefeenheid vervangen (maar beide GRTS-rangnummers worden behouden in de databank). Aansluitend wordt een celkartering uitgevoerd in de vervangcel
```

>         -   Indien geen geschikte rastercel gevonden wordt, is geen lokale vervanging mogelijk en wordt de steekproefeenheid verwijderd uit de steekproef.

Reviewer 2: ```suggestion
        -   Indien geen geschikte vervangcel gevonden wordt, is geen lokale vervanging mogelijk en wordt de steekproefeenheid verwijderd uit de steekproef.
```

In de eerste twee stappen wordt beoordeeld of het doeltype kan aanwezig zijn en ter plekke actueel aanwezig is in het celmidden of ergens in de cel van de steekproefeenheid.
Doeltypes die *ergens* in de cel dienen voor te komen, zijn:

-   habitattypes: 1320, 2150, 6210_sk, 6430_bz en 7150
-   rbb‐types: brem‐ en gaspeldoornstruweel (rbbsg) en doornstruweel (rbbsp).

Alle andere terrestrische doeltypes (met uitzondering van habitattype 7220) dienen aanwezig te zijn in het celmidden.
Voor de kalktufbronnen (habitattype 7220) is de populatie‐eenheid de volledige habitatvlek (met variabele, doorgaans beperkte oppervlakte), geïdentificeerd met een punt gekoppeld aan de vlek.

> De evaluatie van dit doeltype gebeurt dus niet op niveau van rastercellen maar de hele vlek.

Reviewer 2: ```suggestion
De evaluatie van dit doeltype gebeurt dus niet op niveau van rastercellen maar de hele vlek en valt dus buiten het bestek van voorliggend protocol.
```

**Let op**: een steekproefeenheid kan voor meer dan één doeltype zijn getrokken.
Dit is het gevolg van het feit dat een polygoon in de Habitatkaart, waarop de waarden van de rastercellen zijn gebaseerd, meer dan één type (habitat of rbb) kan bevatten, in relatieve verhoudingen zoals weergegeven met de zogenaamde phab-waarden.

> In het celmidden kan er echter normaliter maar één type voorkomen en dus sluiten types die celmiddengebaseerd zijn elkaar uit.

Reviewer 2: ```suggestion
In het celmidden zal er echter in realiteit normaliter maar één type voorkomen en dus sluiten types die celmiddengebaseerd zijn elkaar uit.
```

Wel kan een steekproefeenheid meerdere celgebaseerde types bevatten, al dan niet in combinatie met een celmiddengebaseerd type.
Check dus bij de evaluatie van een terrestrische steekproefeenheid of je deze procedure moet doorlopen voor meer dan één doeltype binnen hetzelfde jaar, dan kan je het werk hiervoor combineren in één procedure.

## Desktopscreening van steekproefeenheden en zijn vervangeenheden {#desktop}

Om grote, duidelijke fouten in de steekproef op te sporen, dient een steekproefeenheid en zijn steekproefeenheden onderworpen te worden aan een eerste visuele screening.

> Hiervoor dient de rastercel van de steekproefeenheid ruimtelijk gevisualiseerd te worden (vb. met QGIS) in combinatie met een aantal GIS-lagen in de meest recente versie, zoals orthofoto's (zomer- en winteropnames), de Habitatkaart, de Watervlakkenlaag en het Digitaal Hoogtemodel Vlaanderen.

Reviewer 2: ```suggestion
Hiervoor dient de rastercel van de steekproefeenheid ruimtelijk gevisualiseerd te worden (vb. in QGIS) in combinatie met een aantal GIS-lagen in de meest recente versie, zoals orthofoto's (zomer- en winteropnames), de Habitatkaart, de Watervlakkenlaag en het Digitaal Hoogtemodel Vlaanderen.
```

Duid aan in de databank met steekproefeenheden of de steekproefeenheid volgens deze screening wel of niet tot het doeltype kan behoren.
Als het er niet toe kan behoren, controleer de vervangeenheden.
Het resultaat van de screening wordt gebruikt om de steekproef bij te werken: indien de steekproefeenheid en zijn vervangeenheden **overduidelijk** niet tot het doeltype kunnen behoren, wordt het verwijderd uit de steekproef.
Indien er geen duidelijke visuele aanwijzingen zijn dat het doeltype er niet kan aanwezig zijn, wordt de steekproefeenheid in een volgende stap beoordeeld op de aanwezigheid van het doeltype ter plekke.

## Beoordeling op aanwezigheid van het doeltype ter plekke {#terplekke}

### Bereid het bezoek aan de steekproefeenheid voor

Controleer in de GIS-applicatie wie de eigenaar en/of beheerder is van het gebied waarin de steekproefeenheid gelegen is.
Toets af bij de meetnetverantwoordelijke of projectcoördinator of vergunningen noodzakelijk zijn en je bezoek is gemeld bij de eigenaar en/of beheerder.

> Toegankelijkheid van militaire gebieden dient een week voor het terreinbezoek aangevraagd te worden via mail aan het betreffende kwartier, met de boswachter in cc.

Reviewer 2: wie doet dit?

Author: de uitvoerder van de locatie-evaluatie

Zij laten weten op welke dagen en tussen welke uren je het terrein mag betreden.
Bij het betreden en het verlaten van het terrein op de dag zelf dien je telkens telefonisch contact op te nemen met de kwartiercommandant.

### Evalueer de timing van je bezoek

Evalueer of je geplande bezoek gebeurt in de (sub)optimale periode voor kartering van het doeltype.

<!-- nog tabel invoegen met optimale periode -->

### Evalueer de toegankelijkheid en bezoek de steekproefeenheid

> Is het terrein waar de steekproefeenheid gelegen is ontoegankelijk, duid dit aan in de GIS-applicatie en vermeld de reden en de vermoedelijke duur van ontoegankelijkheid.

Reviewer 2: ```suggestion
Indien het terrein, waar de steekproefeenheid gelegen is, ontoegankelijk is, duid dit aan in de GIS-applicatie en vermeld de reden en de vermoedelijke duur van ontoegankelijkheid.
```

Mogelijke redenen zijn: overstroming, afsluiting, broedseizoen, beheerwerken, enz.

> Is het terrein wel toegankelijk, navigeer je naar het steekproefeenheid.

Reviewer 2: ```suggestion
Is het terrein wel toegankelijk, navigeer je naar de steekproefeenheid.
```

### Bezoek de steekproefeenheid en navigeer naar het celmidden

Met de hulp van de GPS-ontvanger op je tablet of smartphone navigeer je naar de locatie van de te beoordelen steekproefeenheid.
Eens je je in de nabijheid van de meetlocatie bevindt (vb. op 10 m), navigeer je met behulp van de RTK-GPS-ontvanger naar de exacte locatie van het centrum van de steekproefeenheid.

**Let op**: in de rastercel kan een permanent kwadraat van het meetnet biotische habitatkwaliteit aanwezig zijn.

> Het is 3 m op 3 m groot en bevindt zich ten noordwesten van het celmidden: het celmidden van de rastercel is het zuidoostelijk hoekpunt van het permanent kwadraat.

Reviewer 2: figuurtje toevoegen

Vermijd zo veel mogelijk dit kwadrant om vertrappeling van de vegetatie te voorkomen.
Dit kan eenvoudig door het celmidden te benaderen vanuit het zuidoosten, indien het terrein dit toelaat.

### Evalueer of het doeltype aanwezig is {#doeltype}

De GIS-applicatie geeft aan welk doeltype op een steekproefeenheid aanwezig zou moeten zijn en indien van toepassing de laatste keer dat het type werd bepaald in het veld.
De wijze waarop het wordt geëvalueerd is afhankelijk van het doeltype:

-   voor de evaluatie van de aanwezigheid van habitat(sub)types 1320, 2150, 6210_sk, 6430_bz en 7150 en de rbb‐types brem‐ en gaspeldoornstruweel (rbbsg) en doornstruweel (rbbsp) is het voldoende dat het doeltype ergens in de 32 m x 32 m rastercel aanwezig is.
-   voor alle andere terrestrische types (behalve 7220) dient het doeltype aanwezig te zijn in het midden van de rastercel.

> -   specifiek voor type

Reviewer 2: onvolledig, vermoedelijk gaat het hier over 7220??

Author: mag weg

De bepaling van het aanwezige habitat(sub)type of rbb gebeurt met behulp van de [veldsleutels](https://www.vlaanderen.be/inbo/de-biologische-waarderingskaart/bwk-handleiding-veldsleutels/).

> Het doeltype moet op minstens 9 m^2^ aanwezig zijn binnen de rastercel, uitgezonderd bij habitattype 7150, daar wordt de ondergrens van 4 m^2^ gehanteerd.

Reviewer 2: waarom afwijken voor en alleen voor 7150??

Author: was oorspronkelijk omwille van voorwaarde 4 m² in heidesleutel; zullen we aanpassen naar 9 m², zodat gelijk is voor alle types

> Door veranderingen op het terrein, onnauwkeurigheden in de afbakening van de types of het voorkomen van complexen in de karteringen kan het type afwijken van het type op de Habitatkaart.

Reviewer 2: ```suggestion
Door veranderingen op het terrein, onnauwkeurigheden in de afbakening van de types of het voorkomen van complexen in de karteringen kan het in de praktijk voorkomende type afwijken van het type op de Habitatkaart.
```

Bijgevolg dienen we steeds het type te bepalen met behulp van de veldsleutels.
Het schaalniveau waarop de sleutel dient toegepast te worden, is niet dat van het celmidden, maar deze van de min of meer homogene vegetatievlek waarin het celmidden zich bevindt.

> Om de finale beslissing te maken of het punt wordt meegenomen in de steekproef, wordt een denkbeeldig 3 m x 3 m-proefvlak beoordeeld ten noordwesten van het celmidden.

Reviewer 2: dit toch alleen van celmiddengekoppelde types veronderstel ik?

Reviewer 2: en waarom checken we een denkbeeldige 9m² ten NW van centrum? ik zou logischerwijs denken gewoon rondom centroid van de rastercel (door die NW ga je mogelijks ook weer verstoring van de MHQ plots in de hand werken...)

Author: stel dat steekproefeenheid ook opgevolgd moet worden voor LSVI-steekproef, dan is het relevant te checken of het type aanwezig is in die 9m² ten NW van centrum. als enkel voor abiotiek en LSVI-kartering, is dit niet relevant.

> **Onderstaande beslisregels worden gehanteerd voor het bepalen of de steekproefeenheid al dan niet geschikt is.**

Reviewer 2: gaat dit nu enkel over die denkbeeldige 9m² of over de volledige rastercel???

Reviewer 2: en dus enkel van toepassing voor de centroidgebaseerde types lijkt me

> -   De rastercel maakt deel uit van een homogene, grotere vegetatievlek van het doeltype (minimum 9 m^2^) -\> de steekproefeenheid is geschikt.

Author: aaneengesloten

> -   De rastercel bevat naast het doeltype ook (een) ander habitattype en zijn ruimtelijk duidelijk van elkaar gescheiden (bv. bos naast een heideterrein, ven)

Reviewer 2: opnieuw onduidelijk wegens eerder vermelde regels, indien het centroid in 10m2 doelvegetatie ligt dan doet het aandeel meer of minder dan 50% er toch niet meer toe??

Author: tweede lijn weg
op eerste lijn: rastercel => celmidden

>     -   Het doeltype neemt minder dan 50% van de rastercel in -\> de steekproefeenheid is ongeschikt.

Author: mag weg

>     -   Het doeltype neemt 50% of meer in van de rastercel -\> de steekproefeenheid is geschikt.

Author: mag weg

Geef in de GIS-applicatie de volgende gegevens in:

-   je naam
-   de datum waarop je deze evaluatie uitvoert
-   het actueel aanwezige habitat(sub)type of rbb in de cel of in het celmidden
-   eventuele opmerkingen
-   eventueel een foto.

Is de steekproefeenheid niet geschikt, geef dan ook aan dat een lokale vervanging nodig is en geef de volgende gegevens in:

-   de reden van de nodige vervanging, indien je dit kan inschatten: is het een ander habitattype of rbb (habitatwijziging) of is het geen habitat of rbb meer (habitatverlies);
-   een inschatting of het permanent is of niet.

Is de steekproefeenheid **geschikt**, voer aansluitend een markering en celkartering uit (zie \@ref(celkartering)).
Is de steekproefeenheid **niet geschikt**, start aansluitend met de vervangprocedure (zie \@ref(lokalevervanging)).

## Lokale vervanging van niet-geschikte steekproefeenheden {#lokalevervanging}

Blijkt, na het doorlopen van bovenstaande stappen, dat de steekproefeenheid **het doeltype niet bevat**, dan doorloop je aansluitend de procedure voor het vinden van een geschikte vervangeenheid.

> Je gaat hierbij in (een deel van) de polygoon waarin de niet-geschikte steekproefeenheid gelegen is, op zoek naar een vervangeenheid dat wel het doeltype bevat.

Reviewer 2: ```suggestion
Je gaat hierbij in (een deel van) de polygoon waarin de niet-geschikte steekproefeenheid gelegen is, op zoek naar een vervangeenheid die wel het doeltype bevat.
```

In de GIS-applicatie vind je voor elke te evalueren steekproefeenheid een lijst van mogelijke vervangeenheden die je volgens een vooraf vastgelegde volgorde evalueert zoals beschreven in protocol [sfp-001-nl 2023.01](https://inbo.github.io/protocols/2023.01/index.html).

Aanvullend op dat protocol geldt voor lokale vervangingen in grote polygonen een extra regel.
In het geval van grote polygonen kan het aflopen en evalueren van de mogelijke vervangeenheden zeer veel moeite en tijd kosten.
Om dit te beperken, worden grote polygonen opgedeeld volgens de met de polygoon overlappende GRTS-level03-cellen van 128 m op 128 m.
Enkel die GRTS-level03-cel waarin de originele steekproefeenheid gelegen is (én meer dan ... rastercellen bevat), wordt verder beschouwd.
Deze ruimtelijke beperking van vervanging bij grote habitatpolygonen is reeds doorgevoerd in de lijst van mogelijke vervangeenheden in de GIS-applicatie.

De mogelijke vervangeenheden worden pas gevisualiseerd in de GIS-applicatie wanneer je hebt aangeduid dat een lokale vervanging nodig is en die gegevens hebt opgeslagen (zoals vermeld in \@ref(terplekke)).

> De mogelijke vervangeenheden zijn genummerd en dienen in die volgorde zoals beschreven in het protocol geëvalueerd te worden tot er een vervanging mogelijk is.

Reviewer 2: ```suggestion
De mogelijke vervangeenheden zijn genummerd en dienen in die volgorde geëvalueerd te worden tot er een vervanging mogelijk is.
```

De evaluatie gebeurt zoals in \@ref(doeltype).
In de GIS-applicatie duid je bij het in volgorde evalueren van de vervangeenheden telkens aan of een vervangeenheid het doeltype bevat.
Voldoet een eerstvolgende vervangeenheid, dan vink je dit aan in de GIS-applicatie.
Op dat moment stopt de vervangprocedure; sla de gegevens op in de GIS-applicatie.

> Werd een lokale vervangeenheid gevonden, voer dan de celkartering en markering uit zoals hiervoor beschreven in \@ref(celkartering).

Reviewer 2: ```suggestion
Eens een lokale vervangeenheid gevonden, voer dan de celkartering en markering uit zoals hiervoor beschreven in \@ref(celkartering).
```

## Markering en celkartering van geschikte steekproefeenheden {#celkartering}

Blijkt dat de steekproefeenheid of zijn vervangeenheid **het doeltype bevat**, dan voer je aansluitend een celkartering en markering uit.
De celkartering houdt in dat je ter plaatse beoordeelt waar het doeltype actueel aanwezig is in de 32 m op 32 m rastercel.

> ### Plaats een markering

Reviewer 2: suggestie: foto toevoegen

Plaats met behulp van de hamer een metalen buisanker (van 30 of 50 cm lengte, afhankelijk van het type ondergrond) in de grond op de plaats van het celmidden.
Zorg ervoor dat de bovenkant van het anker nog zichtbaar is en maak hieraan het plastic meetmerk vast.
Dit doe je door het meetmerk op de opening van het anker te plaatsen en de plug door het gat van het meetmerk in het buisanker te slaan.
Hierdoor knelt het meetmerk vast in het buisanker.
Controleer of het anker vast zit in de grond: als je het gemakkelijk kan uittrekken, plaats dan een langer buisanker.
Staat er hoge, dichte vegetatie waardoor je het meetmerk moeilijk zou terugvinden (vb. bramenstruweel, riet), plaats een bamboestok met gekleurde top naast het meetmerk.

### Oriënteer jezelf en de rastercel op het terrein

Oriënteer jezelf op het terrein met behulp van de locatie van het celmidden, de visuele voorstelling van de rastercel en de Habitatkaart in de GIS-applicatie.
Recente orthofoto's kunnen ook hierbij helpen.
Je kan ook navigeren naar (één van) de vier hoekpunten van de rastercel om het gebied binnen de rastercel af te bakenen.

> Dit kan ook letterlijk met behulp van bamboestokken.

Reviewer 2: ```suggestion
Gebruik eventueel 4 bamboestokken om de hoekpunten te markeren
```
